### PR TITLE
fixed model try catch and also got every model to 100% coverage

### PR DESCRIPTION
**Pull request by Martinbalke**

### CODE REVIEW

File: __tests__/database.test.js [hunk 1]
```diff
@@ -38,11 +40,22 @@ let users = {
     password: 'sampassword',
   },
 };
+
+let messageData = {
+  message: 'Wow this app is cool',
```
Comment (CaptainMorganCodes) [inline __tests__/database.test.js:45]: LOL! Very true =)

**Review by CaptainMorganCodes — COMMENTED** [review]
What was the reason for removing the try/catch in create, read, & delete? Are we accounting for this elsewhere? Or is it built in by mongoose?

File: __tests__/database.test.js [hunk 2]
```diff
@@ -4,8 +4,10 @@ const mongoose = require('mongoose');
 const MongoMemoryServer = require('mongodb-memory-server').default;
 const User = require('../model/user/user-model');
 const userDB = new User();
-// const Chat = require('../model/chat/chat-model');
-
+const Chat = require('../model/chat/chat-model');
+const chat = new Chat;
```
Comment (james-ad) [inline __tests__/database.test.js]: Does this need to be `new Chat()`?

Comment (james-ad) [reply __tests__/database.test.js]:
```suggestion
const chat = new Chat();
```

File: __tests__/database.test.js [hunk 3]
```diff
@@ -4,8 +4,10 @@ const mongoose = require('mongoose');
...
+const Connection = require('../model/connection/connection-model');
+const connection = new Connection;
```
Comment (james-ad) [inline __tests__/database.test.js]:
```suggestion
const connection = new Connection;
```
```suggestion
const connection = new Connection;
```
```suggestion
const connection = new Connection();
```

**Review by james-ad — CHANGES_REQUESTED** [review]

**Review by CaptainMorganCodes — APPROVED** [review]

**Review by james-ad — APPROVED** [review]

**Pull request merged**